lib/rules/no-restricted-globals.js

```diff
@@ -16,10 +22,34 @@ const TYPE_NODES = new Set([
 	"TSQualifiedName",
 ]);
 
+const GLOBAL_OBJECTS = new Set(["global", "globalThis", "self", "window"]);
+
 //------------------------------------------------------------------------------
 // Rule Definition
 //------------------------------------------------------------------------------
 
+const arrayOfStringsOrObjects = {
```

Comment:
Maybe `arrayOfGlobals` to be more specific? (Took me a second to realize this wasn't just a generic structure.)